src/saving/SaveListItem.cs

```diff
@@ -172,6 +177,8 @@ public override void _Process(float delta)
         createdOnPlatform.Text = save.Info.Platform;
         description.Text = save.Info.Description;
 
+        isBroken = save.Info is BrokenSaveInformation;
```

Comment:
This could use the save info type to determine if it is broken, instead of needing to rely on a subclass existing.